<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailPreviewer</class>
 <widget class="QWidget" name="EmailPreviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="m_mainLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_tbSubject">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>From</string>
     </property>
     <property name="buddy">
      <cstring>m_tbFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_tbFrom">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnAttachments">
       <property name="text">
        <string>&amp;Attachments</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnReply">
       <property name="text">
        <string>&amp;Reply</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnForward">
       <property name="text">
        <string>&amp;Forward</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Subject</string>
     </property>
     <property name="buddy">
      <cstring>m_tbSubject</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_tbTo">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>To</string>
     </property>
     <property name="buddy">
      <cstring>m_tbTo</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_tbFrom</tabstop>
  <tabstop>m_tbTo</tabstop>
  <tabstop>m_tbSubject</tabstop>
  <tabstop>m_btnReply</tabstop>
  <tabstop>m_btnForward</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
